### PR TITLE
Add occupancy sensor to WiZ

### PR DESCRIPTION
**Pull request by bdraco**
## Proposed change

Add occupancy sensor to WiZ

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/66231
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by bdraco** [comment]
<p>Discourse isn’t available right now. We are currently upgrading the site, or performing scheduled maintenance.</p>

File: source/_integrations/wiz.markdown [hunk 1]
```diff
@@ -2,19 +2,30 @@
...
   - "@sbidy"
 ha_domain: wiz
 ha_platforms:
+  - binary_sensor
   - light
   - switch
 ---
 
-The WiZ integration allows you to control your WiZ lights and smart sockets.
+The WiZ integration allows you to control your WiZ lights, occupancy sensors, and smart sockets.
```
Comment (bdraco) [inline source/_integrations/wiz.markdown]:
```suggestion
The WiZ integration allows you to control your WiZ lights and smart sockets.

The integration can report the state of occupancy sensors that have been linked to a device.
```

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by frenck** [comment]
I think we need to revert the last commit?

**Comment by bdraco** [comment]
Works better if I push the revert to `upstream` instead of my remote

**Review by frenck — APPROVED** [review]

**Pull request merged**